internal/pkg/v1/action/catalog_create.go

```diff
@@ -7,16 +7,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	olmv1 "github.com/operator-framework/operator-controller/api/v1"
-)
 
-type createClient interface {
-	creator
-	deleter
-	getter
-}
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
 
 type CatalogCreate struct {
-	client         createClient
+	config         *action.Configuration
```

Comment:
We should change it to `Config` instead of `config` to make it consistent with other members of the struct.

Reply:
You are right, this is fine.